src/v2/Apps/Shows/Routes/ShowsCity.tsx

```diff
@@ -48,6 +57,44 @@ export const ShowsCity: React.FC<ShowsCityProps> = ({ viewer, city }) => {
     )
   }, [city.upcomingShows])
 
+  const [loading, setLoading] = useState(false)
+
+  const currentShowsRef = useRef<HTMLDivElement | null>(null)
+  const { scrollTo } = useScrollTo({
+    selectorOrRef: currentShowsRef,
```

Comment:
Should maybe rename to `element`? 🤷

Reply:
I like selector or ref 👍 

Thoughts on moving this to Palette?

Reply:
It's specific to Force, since it's specific to our header and our sticky implementation. Maybe everything can go over there once we remove hard-coding, but I don't necessarily think it'd be that useful as part of a UI package. Maybe a separate `palette-hooks` package or something down the line.